src/scripts/domsdatabasen_example_usage.py

```diff
@@ -0,0 +1,9 @@
+"""Example usage of the Domsdatabasen API"""
+
+from alexandra_ai_data.domsdatabasen import Domsdatabasen
+
+domsdatabasen = Domsdatabasen()
+
+case = domsdatabasen.get_case(case_id="100")
+
+print(case)
```

Comment:
Consider moving these kinds of examples to a separate `examples` directory, in the root folder.

Alternatively, you could simply add this bit of code to the readme under the Quick Start section.